packages/subgraph/schema.graphql

```diff
@@ -48,6 +48,7 @@ type IpfsMetadata @entity {
   prompts: [String!]!
   communicationType: String!
   communicationURL: String!
+  communicationParams: JSON
```

Comment:
i don't think subgraph accepts JSON type, we might have to split them into two values communicationParamsURL and communicationParamsID

Reply:
I see, could you check please what's the best way to implement it